Sources/App/Models/Entities/AppUser.swift

```diff
@@ -0,0 +1,38 @@
+import Bugsnag
+import FluentKit
+import Vapor
+
+final class AppUser: Authenticatable, Model {
+    static let schema = "AppUsers"
+
+    @ID(custom: "id", generatedBy: .user)
+    var id: String?
+
+    @Field(key: "hashedPassword")
```

Comment:
Please have a look at our FieldKey extension and how it is used. It also affects the migrations.